client/components/foldable-card/index.jsx

```diff
@@ -56,6 +55,26 @@ class FoldableCard extends Component {
 		}
 	}
 
+	clickAction = () => ! this.props.clickableHeader && ! this.props.disabled && this.onClick();
+
+	clickExpander = () => ! this.props.clickableHeader && ! this.props.disabled && this.onClick();
+
+	clickHeader = () => this.props.clickableHeader && ! this.props.disabled && this.onClick();
+
+	onActionKeyPress = event => {
+		// enter press
+		if ( 13 === event.keyCode ) {
```

Comment:
You can get rid of `onActionKeyPress` & `onHeaderKeyPress` if you follow my other suggestions, but for future reference, [buttons are triggered on both enter and space](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/#keyboard-interaction-3), so you'd want to check for both ([see Focusable](https://github.com/Automattic/wp-calypso/blob/master/client/components/focusable/index.jsx#L23)).